package.json

```diff
@@ -33,7 +33,8 @@
     "sse": "0.0.6",
     "strong-error-handler": "^1.0.0",
     "traverse": "^0.6.6",
-    "xml2js": "^0.4.8"
+    "xml2js": "^0.4.8",
+    "strong-globalize": "^2.6.2"
```

Comment:
would prefer to see it alphabetical order 
`npm install --save strong-globalize` would make it alphabetical order